README.md

```diff
@@ -124,8 +124,9 @@ Check queries.sql for examples of queries that can be run on a newly created dat
 
```

Comment:
![Screenshot from 2023-04-11 20-54-22](https://user-images.githubusercontent.com/93039359/231219923-5acc2620-ce9c-4a67-a4a5-aecae2b971c9.png)

**[OPTIONAL]**

- [ ] It would be very nice if you make all your commit messages capitalize to make your readme file descriptive enough :+1:

Reply:
thank you @HAMMAS-SALEEM !